component-runtime-impl/src/main/java/org/talend/sdk/component/runtime/input/Streaming.java

```diff
@@ -176,6 +176,12 @@ public interface StopStrategy {
 
         boolean shouldStop(long read);
 
+        long getMaxReadRecords();
+
+        long getMaxActiveTime();
+
+        long getStarted();
```

Comment:
getStartedTime() Because I couldn't get what getStarted is. I even thing that we should change to isStarted but then I saw that it returns long. And I kept looking

Reply:
yep, that's a bit misleading! In fact, it means _lifecycle started at time n (system millis)_... Will refactor to get a better understanding like `getStartedAtTime`.